MapboxCoreNavigation/PortableRouteController.swift

```diff
@@ -94,3 +378,70 @@ open class PortableRouteController: RouteController {
         return navigator.getHistory()
     }
 }
+
+extension PortableRouteController: Router {
+    public func userIsOnRoute(_ location: CLLocation) -> Bool {
```

Comment:
This appears to be the same as `userIsWithinRadiusOfRoute()` [above](https://github.com/mapbox/mapbox-navigation-ios/pull/1904/files#diff-1dff2075b809825263a18d19396f5c0aR343)